config/devices/default-youview-default.json

```diff
@@ -0,0 +1,131 @@
+{
+    "pageStrategy": "html5hbbtvhybrid",
+    "modules": {
+        "base": "antie/devices/browserdevice",
+        "modifiers": [
+            "antie/devices/anim/noanim",
+            "antie/devices/media/html5memoryleakfix",
+      "antie/devices/mediaplayer/html5",
+      "antie/devices/mediaplayer/live/seekable",
+            "antie/devices/net/default",
+            "antie/devices/data/nativejson",
+            "antie/devices/broadcastsource/hbbtvsource",
+            "antie/devices/storage/cookie",
+            "antie/devices/logging/consumelog",
+            "antie/devices/exit/closewindow",
+            "antie/devices/parentalguidance/youviewpghandler"
+        ]
+    },
+    "logging": {
+        "level": "none"
+    },
+    "statReporters": [
+        "tvpjslib/stats/avreporters/youviewreporter"
```

Comment:
This is interesting - they've added this "statReporters" thing in here. They did mention during the visit that YouView requires some custom stats reporting. It's worthing keeping an eye on this :)